website/templates/project/registrations.mako

```diff
@@ -127,12 +127,11 @@
 </div>
 </div>
 <script type="text/html" id="createDraftRegistrationModal">
-    <p>Registration creates a frozen version of the project that can never be edited or deleted but can be withdrawn. Your original project remains editable but will now have the registration linked to it. Things to know about registration:</p>
+    <p>Registration creates a frozen version of the project. Your original project remains editable and will have the registration linked. Things to know about registration:</p>
     <ul>
-        <li>Ensure your project is in the state you wish to freeze before registering.</li>
-        <li>Consider turning links into forks.</li>
-        <li>Registrations can have embargo periods for up to four years. If you choose an embargo period, the registration will automatically become public when the embargo expires.</li>
-        <li>Withdrawing a registration removes the contents of the registrations but will leave behind a log showing when the registration was created and withdrawn.</li>
+        <li><b>Registrations cannot be edited or deleted.</b></li>
```

Comment:
Not a blocker (I should've noticed this in first pass), but always prefer `<strong>` to `<b>`: https://stackoverflow.com/questions/271743/whats-the-difference-between-b-and-strong-i-and-em/271776#271776.